src/ArticlePage.jsx

```diff
@@ -8,10 +13,19 @@ function ArticlePage() {
   const [isLoading, setIsLoading] = useState(true);
   const [article, setArticle] = useState({});
   const [comments, setComments] = useState([]);
+  const [articleVotes, setArticleVotes] = useState(0);
+
+  function handleClick(vote) {
+    setArticleVotes((articleVotes) => articleVotes + vote);
+    changeVote(article_id, vote).catch((err) => {
```

Comment:
In this case the .catch block is redundant. As the errors are handled and caught in the back end and are sent back as messages, the error would be on the response object. Therefore the way to handle these would be to check the message we get back from the patch, if it is the data, it is successful, if it is a string of an error message, then that is how we would get the error on the front end. We need to check the response object :)